segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/FileSystemIntegrationTest.java

```diff
@@ -31,7 +31,7 @@
 public class FileSystemIntegrationTest extends StreamSegmentStoreTestBase {
     //region Test Configuration and Setup
 
-    private static final int BOOKIE_COUNT = 3;
+    private static final int BOOKIE_COUNT = 1;
```

Comment:
Why are you dropping the number of bookies?

Reply:
There is no need to have 3 bookies running during unit tests (in-memory, single disk). This was bothering me to a point where I decided to drop it down to 1 (more bookies -> slower tests). We are not testing BK failover here, so no point in having more.

Reply:
Reverted.